pkg/symbolizer/symbolizer_test.go

```diff
@@ -411,23 +412,32 @@ func setup(t *testing.T) (*grpc.ClientConn, *debuginfo.Store, metastore.ProfileM
 	sym, err := symbol.NewSymbolizer(logger)
 	require.NoError(t, err)
 
+	cfg := &debuginfo.Config{
+		Bucket: &client.BucketConfig{
+			Type: client.FILESYSTEM,
+			Config: filesystem.Config{
+				Directory: "testdata/",
+			},
+		},
+		Cache: &debuginfo.CacheConfig{
+			Type: debuginfo.FILESYSTEM,
+			Config: &debuginfo.FilesystemCacheConfig{
+				Directory: cacheDir,
+			},
+		},
+	}
+
+	httpDebugInfodClient, err := debuginfo.NewHTTPDebugInfodClient(logger, "https://debuginfod.systemtap.org", 4*time.Millisecond)
```

Comment:
4 milliseconds timeout seems very very low for a download that might be several 100s of megabytes large. I would rather make this something like 2 or 5 minutes.

Reply:
so, I should change the default value for the `DebugInfodHTTPRequestTimeout` flag as well?

Reply:
I think this is just for the tests, no?

Reply:
oh whops my bad I don't know how I didn't see that this was a test